PocketKit/Sources/Analytics/Entities/ContentEntity.swift

```diff
@@ -17,25 +17,13 @@ public struct ContentEntity: Entity {
      */
     let url: URL
 
-    /**
-     * The backend identifier for a URL.
-     */
-    let itemId: String?
-
-    public init(url: URL, itemId: String? = nil) {
+    public init(url: URL) {
         self.url = url
-        self.itemId = itemId
```

Comment:
Per https://pocket.slack.com/archives/CQKRXK2G6/p1677095703451209?thread_ts=1677088316.945709&cid=CQKRXK2G6 I am removing this.